protocol/rpcconsumer/rpcconsumer_server.go

```diff
@@ -855,6 +854,8 @@ func (rpccs *RPCConsumerServer) relayInner(ctx context.Context, singleConsumerSe
 			}
 			return reply, 0, err, backoff
 		}
+		analytics.SetProcessingTimestamp(time.Now())
```

Comment:
you can use this method to set the true / false. meaning if we got here the relay was successful and we can measure on the way back. otherwise the relay was a failure and we do not measure way back.